presto-main/src/main/java/com/facebook/presto/sql/gen/CursorProcessorCompiler.java

```diff
@@ -87,7 +96,13 @@ public void generateMethods(ClassDefinition classDefinition, CallSiteBinder call
         constructorBody.comment("super();")
                 .append(thisVariable)
                 .invokeConstructor(Object.class);
+
         cachedInstanceBinder.generateInitializations(thisVariable, constructorBody);
+        for (PreGeneratedExpressions preGeneratedExpressions : allPreGeneratedExpressions) {
+            preGeneratedExpressions.getLambdaFieldMap().values().forEach(
+                    lambdaExpressionField -> lambdaExpressionField.generateInitialization(thisVariable, constructorBody)
```

Comment:
Use a for-loop here instead. I don't have a particularly strong opinion. But I think it's easier to read that way.